File: concepts/data-types.py
# Data Types
""" 
String
Int
Float

None

Array
    List
    Tuples
    Set

Dict

"""
""" 
var_name = value
"""

nombre = "John"
apellido = 'Doe'
nombre_completo = f"{nombre} {apellido}"

edad = 42
precio = 10.50
temp = -10.5

users = None
posts = None

# Lista

numeros = [1, 2, 3, 4]
nombres = list()
nombres.append("Hugo")
nombres.append("Paco")
nombres.append("Luis")

nombres[0]
numeros.append(5)
numeros.pop()

# Tuples
status = ("active", "inactive", "suspended", "canceled", "completed")
status[0]

# Sets
frutas = {"Pera", "Manzana", "Uva", "Banana", "Kiwi", "Durazno"}

print(nombres)

print(dir(numeros))

# Dict

persona = {
    "nombre": "John",
    "apellido": "Doe"
}

print(persona["nombre"])

persona["edad"] = 10

print(persona)

File: concepts/data-types.js
// Data Types
/* 

String
Number
Boolean
Undefined

Object: 
    Array
    Null
    Literal Object

*/

/* let const var */
/* 
let varName = value;
*/

let nombre = "John";
let apellido = 'Doe'
let nombreCompleto = `${nombre} ${apellido}`

let edad = 42
let precio = 10.50
let temp = -5.5

let users = null;
let posts;

let numeros = [1, 2, 3, 4]
//let nombres = array("Hugo", "Paco", "Luis")

//nombres[0]
numeros.push(5)
numeros.pop()

console.log(numeros)
console.log(typeof numeros)

let persona = {
    nombre: 'John',
    apellido: 'Doe'
}

console.log(persona.nombre)
console.log(persona["apellido"])

persona["edad"] = 0

console.log(persona)